### ISSUE TITLE
reghdfe postestimation influence

### ISSUE THREAD

**Issue by laszlosandor** [sergiocorreia/reghdfe#195]
I was wondering how much of this could work with `reghdfe`:
https://www.stata.com/features/overview/linear-regression-and-influence/

Currently (v 5.7.2) `dfbeta` surely does not.

FWIW, I was hoping to learn where my identification is coming from. Not just on the conceptual level but observation-by-observation.

**Comment by sergiocorreia** [comment]
Dfbeta appears to require one regression per observation (kinda like leave one out regression). That's a bit insane from a computational perspective and for the usual dataset size of reghdfe, so I don't think it's practical.

There might be a few ways to speed up the leave-one-out computations, but they would involve a lot of code.

**Comment by laszlosandor** [comment]
Yeah, I'm not sure what I was hoping for. They surely have no other general-purpose solution other than looping through all observations and if them out one by one.

OK, other than that, is there a reasonable way to detect which observations influence `reghdfe` estimates most? If not for each coefficient separately, at least for overall leverage or something?
